astroquery/gaia/core.py

```diff
@@ -205,12 +432,12 @@ def __query_object(self, coordinate, radius=None, width=None, height=None,
                     ORDER BY
                       dist ASC
                     """.format(**{
-                        'row_limit': "TOP {0}".format(self.ROW_LIMIT) if self.ROW_LIMIT > 0 else "",
-                        'ra_column': self.MAIN_GAIA_TABLE_RA,
-                        'dec_column': self.MAIN_GAIA_TABLE_DEC,
-                        'columns': columns, 'table_name': self.MAIN_GAIA_TABLE,
-                        'ra': ra, 'dec': dec, 'width': widthDeg.value, 'height': heightDeg.value,
-                    })
+                'row_limit': "TOP {0}".format(self.ROW_LIMIT) if self.ROW_LIMIT > 0 else "",
```

Comment:
this new indent doesn't look right, these belong to `format`. But then it may be much better to move all of these out of the string definition to variable to be used as fstrings?